x-pack/plugins/security_solution/public/timelines/components/field_renderers/field_renderers.tsx

```diff
@@ -244,11 +246,14 @@ export const DefaultFieldRendererComponent: React.FC<DefaultFieldRendererProps>
         <EuiFlexItem grow={false}>{draggables} </EuiFlexItem>
         <EuiFlexItem grow={false}>
           <DefaultFieldRendererOverflow
-            rowItems={rowItems}
+            attrName={attrName}
+            fieldType="keyword"
             idPrefix={idPrefix}
-            render={render as OverflowRenderer}
-            overflowIndexStart={displayCount}
+            isAggregatable={true}
             moreMaxHeight={moreMaxHeight}
+            overflowIndexStart={displayCount}
+            render={render as OverflowRenderer}
```

Comment:
nit: we can change the render type definition on DefaultFieldRendererProps to be `render: (item: RowItemTypes) => React.ReactNode;` and then this cast is no longer necessary, and can also just delete type OverflowRenderer. There's also a lot of unused elsewhere stuff that's exported from this file we could get rid of if wanted.